examples/package.json

```diff
@@ -55,6 +55,7 @@
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.6",
     "@types/styled-components": "^5.1.4",
+    "@types/three": "link:../viewer/node_modules/@types/three",
```

Comment:
Hm - it's a bit two-sided. Clients will actually often use ThreeJS, so it makes sense that it's here imo... Not sure how its best handled

Reply:
I interperate this as you wouldn't link threejs like this in client apps. Rather import three js directly

Reply:
So we should rather keep `@types/three` in sync in viewer/ and examples/? What would that give us? Does it make sense to have @types/three be a common dependency so types always are available, just like three?

Reply:
It doesn't give us anything in practice, but at least it mimics our clients' environment. It doesn't matter that much to me though tbh. What do you think @Maksim?

Reply:
Maybe my suggestion about including @types/three is the right way to go about this?

Reply:
> Clients will actually often use ThreeJS

I think the safest way for clients here will be to use THREE from Reveal. Otherwise, there always a chance for some collisions, no matter how to configure three in reveal (as a peer or as a common dependency).

> so it makes sense that it's here imo

I agree it makes sense to have some of our environments closer to clients setup, maybe we should check what will happen for three.js versions mismatch in our examples. But I guess that 0 clients have a link on the exact three.js version that Reveal uses in their package.json. 

> we should rather keep @types/three in sync in viewer/ and examples/

Hm... I would say we shouldn't have them linked at least. Linking is a cheat to have them in sync, but speaking about clients, they probably don't want to sync anything with Reveal internals. So by not doing syncs we at least might discover when they are actually must be done (or things won't work). 

TLDR; I suggest to avoid npm-links from Reveal node_modules in examples & documentations to have a better understanding of what will go wrong in clients projects. So at least we will know whether that sync will be necessary and when. 

@larsmoa @christjt

Reply:
Ok, if we include types/three in Reveal we can remove the links, right? I suggest we do that

Reply:
> if we include types/three in Reveal we can remove the links, right

@larsmoa it depends... If we use THREE only from reveal as `import { THREE } from '@cognite/reveal` then yes, we can remove the links and that's it.

If we (or client app) want to import things from `three` directly, then `three` and `@types/three` still need to be installed. In some cases, Reveal and a client app will have the same version of three installed, and sometimes they won't. And when they won't, in theory, there might be compile-time issues with three.js types when they are not compatible